### PR TITLE
Feature:fix softdelete bug(temporary solution) and add create team validator 

### PR DESCRIPTION
**Pull request by MayueCif**

### CODE REVIEW

File: src/Services/Masa.Auth.Service.Admin/Program.cs [hunk 1]
```diff
@@ -27,6 +29,8 @@
 
 builder.Services.AddMasaRedisCache(builder.Configuration.GetSection("RedisConfig"));
 
+builder.Services.AddScoped<ISaveChangesFilter, SoftDeleteSaveChangesFilter>();
```
Comment (doddgu) [inline src/Services/Masa.Auth.Service.Admin/Program.cs:32]: Why did you add this manually?

File: src/Web/Masa.Auth.Web.Admin.Rcl/wwwroot/i18n/zh-CN.json [hunk 2]
```diff
@@ -116,6 +116,8 @@
   "Team Type": "团队类型",
   "Create Team": "创建团队",
   "Update Team": "编辑团队",
+  "Delete Team": "删除团队",
+  "Confirm Delete Team": "确认删除该团队？",
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Admin.Rcl/wwwroot/i18n/zh-CN.json:120]: Is this a sentence or some words? Why some words like `Confirm Delete Team` and some words like `Failed to add menuPermission`?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Services/Masa.Auth.Service.Admin/Program.cs (hunk 1 above)
Comment (MayueCif) [reply src/Services/Masa.Auth.Service.Admin/Program.cs:32]: to fixed softdelete,when update contrib preview3 will delete this

**Review by doddgu — APPROVED** [review]
LGTM
